src/utils/redux.js

```diff
@@ -210,31 +210,46 @@ function textCommentCreate(state, action) {
 // }
 
 
-function voiceRecordingListSet(state, action) {
-  const voiceRecordings = action.voiceRecordings,
+function textCommentDelete(state, action) {
+  const textComments = state.textComments.filter(i => i.uuid !== action.textCommentUuid),
+        aggregateFeed = [...textComments, ...state.voiceRecordings],
+        feed = aggregateFeed.sort((a, b) => {
+          return a.timestampMilliseconds - b.timestampMilliseconds;
+        });
+
+  return {
+    ...state,
+    textComments: textComments,
+    feed: feed,
+  }
+}
+
+
+function voiceRecordingCreate(state, action) {
```

Comment:
The diff is super ugly here, but this is the new reducer. It is responsible for transforming the server response into a new voice recording object in the Redux layer.